### ISSUE TITLE
Passing all props to <input> element

### ISSUE THREAD

**Issue by jacobboon** [ericgio/react-bootstrap-typeahead#29]
The latest React release ([15.2.0](https://github.com/facebook/react/blob/master/CHANGELOG.md), from July 1st) produces a warning that not all props should be passed to the <input> element.

```
[Error] Warning: Unknown props `filteredOptions`, `labelKey`, `onAdd`, `onRemove`, `text` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by TypeaheadInput)
    in div (created by TypeaheadInput)
    in TypeaheadInput (created by Typeahead)
    in div (created by Typeahead)
    in Typeahead (created by TypeaheadClickOutside)
    in TypeaheadClickOutside (created by SearchBarContainer)
```

This seems to be an issue with [`TypeaheadInput.react.js#L44`](https://github.com/ericgio/react-bootstrap-typeahead/blob/master/src/TypeaheadInput.react.js#L44)

**Comment by ericgio** [comment]
Thanks for the report! I'll take a look.

**Comment by ericgio** [comment]
Fixed in v0.6.0
